apps/advisor/pkg/app/checkscheduler/checkscheduler.go

```diff
@@ -175,7 +175,7 @@ func (r *Runner) cleanupChecks(ctx context.Context) error {
 		labels := check.GetLabels()
 		checkType, ok := labels[checks.TypeLabel]
 		if !ok {
-			klog.Error("Check type not found in labels", "check", check)
+			r.log.Error("Check type not found in labels", "check", check)
```

Comment:
I assume this doesn't need the contextual info?

Reply:
I missed this 👍